src/v0/destinations/salesforce/transform.js

```diff
@@ -217,7 +217,9 @@ async function getSalesforceIdFromPayload(
     // its a lead object. try to get lead object id using search query
     // check if the lead exists
     // need to perform a parameterized search for this using email
-    const email = getFieldValueFromMessage(message, "email");
+    const email = encodeURIComponent(
+      getFieldValueFromMessage(message, "email")
```

Comment:
did we test with saleforce using real data?

Reply:
Yes